### DISCUSSION TITLE
how to remove an ad area

### DISCUSSION THREAD

**Discussion by kmw0410** [AdguardTeam/AdGuardHome#7142] [Q&A]
i set up adguard home and the ads are removed, but the ad area is till there
what else do i need to set?
![image](https://github.com/user-attachments/assets/4421b603-5d2b-4681-a60b-3f64c1b2ba56)
![image](https://github.com/user-attachments/assets/d64cc7cb-e96f-4a20-9dc5-f6b63abbfd38)

**Comment by vkuz-dev** [comment]
Download ad blocker extension? AdguardHome blocks ads only on DNS level, this means that it only prevents **downloading** ad content, it doesn't have access to the actual web page, thus, some types of ad banners will still be visible like in your case.
